### PR TITLE
Generating the Microsoft.NETCoreSdk.BundledVersions.props file in a new location

### PR DESCRIPTION
**Pull request by livarcocc**
Generating the Microsoft.NETCoreSdk.BundledVersions.props file in the CLI folder, so that it can be picked up by the SDK.

We can remove the other Microsoft.NETCoreSdk.BundledVersions.props file once we do the VS Resolver. If you want to remove it now, let me know, but I wanted to keep it around so that we can insert the CLI in case we need to.

@nguerrera @dotnet/dotnet-cli

### CODE REVIEW

File: build/MSBuildExtensions.targets [hunk 1]
```diff
@@ -63,6 +63,8 @@ Copyright (c) .NET Foundation. All rights reserved.
     <WriteLinesToFile File="$(GeneratedMSBuildExtensionsDirectory)/$(BundledVersionsPropsFolder)/$(BundledVersionsPropsFileName)"
                       Lines="$(BundledVersionsPropsContent)"
                       Overwrite="true" />
-
+    <Copy
```
Comment (eerhardt) [inline build/MSBuildExtensions.targets]: Do you want to `Move` it, or really copy it to 2 different spots?  Why have a duplicated file?

File: build/MSBuildExtensions.targets [hunk 2]
```diff
@@ -63,6 +63,8 @@ Copyright (c) .NET Foundation. All rights reserved.
...
+      DestinationFiles="$(SdkOutputDirectory)/$(BundledVersionsPropsFileName)"
```
Comment (eerhardt) [inline build/MSBuildExtensions.targets]: (nit) usually it goes copy `source` `destination`.  This tripped me up to put the destination first.

File: build/MSBuildExtensions.targets (hunk 1 above)
Comment (livarcocc) [inline build/MSBuildExtensions.targets]: I want to copy for now. See my description of the PR above. We can stop generating the other one later, once we have the resolver change in.

File: build/MSBuildExtensions.targets (hunk 2 above)
Comment (livarcocc) [inline build/MSBuildExtensions.targets]: My editor generated it like that when I hit tab. I will fix it.

**Comment by livarcocc** [comment]
@dotnet-bot Test Debian8.2 x64 Debug Build

**Pull request merged**